apps/dashboard/app/models/batch_connect/session.rb

```diff
@@ -127,6 +127,10 @@ def all
       def find(id)
         new.from_json(db_root.join(id).read)
       end
+
+      def exist(id)
```

Comment:
can this api be `exist?(id)` with a question mark.

Reply:
Name updated